core-android/src/main/java/com/uber/sdk/android/core/auth/LoginActivity.java

```diff
@@ -93,37 +106,71 @@ public static Intent newIntent(
             @NonNull SessionConfiguration sessionConfiguration,
             @NonNull ResponseType responseType) {
 
-        return newIntent(context, sessionConfiguration, responseType, false);
+        return newIntent(context, sessionConfiguration, responseType, "");
     }
 
     /**
      * Create an {@link Intent} to pass to this activity
      *
-     * @param context the {@link Context} for the intent
+     * @param context              the {@link Context} for the intent
      * @param sessionConfiguration to be used for gather clientId
-     * @param responseType that is expected
-     * @param forceWebview Forced to use old webview instead of chrometabs
+     * @param responseType         that is expected
+     * @param isParFlow            specifies whether to send user's profile info to Uber as part of login/signup flow
      * @return an intent that can be passed to this activity
      */
     @NonNull
     public static Intent newIntent(
```

Comment:
These are breaking API changes. The proper way to handle this is to mark as deprecated, and create a new method. This could have more concerns, because the API signature doesn't change, but the runtime behavior does, so there's a runtime risk.

Reply:
Nice catch. I will make the relevant changes